### ISSUE TITLE
Bug: wrong unit in Grafana Volume Size

### ISSUE THREAD

**Issue by rdxmb** [syseleven/syseleven-exporter#61]
prometheus output

```
# HELP syseleven_volume_space_total_gigabytes Quota for volume space per region and project in gigabytes
# TYPE syseleven_volume_space_total_gigabytes gauge
syseleven_volume_space_total_gigabytes{project="e6dc7166ad3d44ddb0a61b3c112cac5e",region="cbk"} 500
syseleven_volume_space_total_gigabytes{project="e6dc7166ad3d44ddb0a61b3c112cac5e",region="dbl"} 0
syseleven_volume_space_total_gigabytes{project="e6dc7166ad3d44ddb0a61b3c112cac5e",region="fes"} 16384
```
![unit](https://github.com/user-attachments/assets/1f70bb7a-df6a-45f1-b096-640cd53ab39f)

**Comment by baurmatt** [comment]
@rdxmb Thanks for reporting! :) This should be fixed with https://github.com/syseleven/syseleven-exporter/pull/64

**Comment by baurmatt** [comment]
Released as chart-0.6.1
